src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java

```diff
@@ -83,9 +82,11 @@ public abstract class AbstractNetworkModificationTest {
     @MockBean
     private NetworkStoreService networkStoreService;
 
-    @Autowired
+    @MockBean
     private ReportService reportService;
 
+    protected ReporterModel reporterModel;
```

Comment:
I'm not sure it's the clean way to do it.
For me you should use Mockito.verify with ArgumentCaptor or ArgumentMatcher.

Reply:
Yes

Reply:
Done

Reply:
Good !